src/platforms/apple/config.yml

```
@@ -4,3 +4,6 @@ caseStyle: canonical
 supportLevel: production
 aliases:
   - cocoa
+categories:
+  - mobile
+  - desktop
```

For watchOS, tvOS we could add wearable and .. iot?

```suggestion
  - desktop
  - wearable
```

---

it'd be an invalid value though, I'd stick to mobile desktop/mobile only

https://github.com/getsentry/sentry-docs/blob/dd7208a0cd9d82fa742a6847fcb1f2c8d4d423ed/src/gatsby/createSchemaCustomization/platformSchema.ts#L16-L22

---

TIL, thanks